rwengine/src/ai/CharacterController.cpp

```diff
@@ -239,6 +239,28 @@ bool Activities::EnterVehicle::update(CharacterObject *character,
         } else if (character->animator->getAnimation(AnimIndexAction) ==
                    anm_enter) {
             if (character->animator->isCompleted(AnimIndexAction)) {
+                if (vehicle->getVehicle()->vehiclename_ == "TAXI"
+                    || vehicle->getVehicle()->vehiclename_ == "CABBIE"
+                    || vehicle->getVehicle()->vehiclename_ == "BORGNINE") {
+                    // Earn $25 from taxi cab
+                    // @todo
+                } else if (vehicle->getVehicle()->vehiclename_ == "POLICAR") {
+                    // Police cruisers give 5 shotgun cartridges
+                    character->addToInventory(4, 5);
+                } else if (vehicle->getVehicle()->vehiclename_ == "ENFORCR") {
+                    // Obtain 100 armour from Enforcer
+                    character->getCurrentState().armour = 100.f;
+                } else if (vehicle->getVehicle()->vehiclename_ == "AMBULAN") {
+                    // Receive up to 25 HPs from Ambulance
+                    if (character->getCurrentState().health < 100.f) {
+                        if (character->getCurrentState().health > 75.f) {
+                            character->getCurrentState().health = 100.f;
+                        } else {
+                            character->getCurrentState().health += 25.f;
+                        }
+                    }
```

Comment:
What happens in VC? [does GTA3 have raised health too?]
Are we certain that this is capped to 100 and not the maximum heath? (150 etc.?)

You can probably solve this easiest by creating a temporary variable and clamping the health before assigning it.
Also the `100.f` should then probably be a variable called `maxHealth` or something (in the player class or something or optionally in this code branch with a `// todo: Replace by correct maxHealth once it is implemented`)

I believe I made a similar mistake in the cheat-handler.